runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -282,6 +288,33 @@ boolean hasOverlay(final Skill skill)
 		return overlayManager.anyMatch(o -> o instanceof XpInfoBoxOverlay && ((XpInfoBoxOverlay) o).getSkill() == skill);
 	}
 
+	void sortByXp()
+	{
+		for (Skill skill : Skill.values())
+		{
+			if (xpState.getSkillSnapshot(skill).getXpGainedInSession() != 0 && skill != Skill.OVERALL)
```

Comment:
Since the xp gained in session is used twice here, store it in a variable instead of calling `getSkillSnapshot()` twice.